### PR TITLE
Feat/117 add cache

### PR DESCRIPTION
**Pull request by Zach13131**

### CODE REVIEW

File: src/accommodation/accommodation.service.ts [hunk 1]
```diff
@@ -809,4 +837,16 @@ export class AccommodationService {
 
     return orderingObjsArray;
   }
+
+  private isDefaultOptions(options: OrderAndFilterDto): boolean {
+    return (
+      (options.limit ?? 12) === 12 &&
```
Comment (gitkoshelew) [inline src/accommodation/accommodation.service.ts]: looks strange isn't enough here 'options.limit === 12 &&' and create const at least for this value, please. looks like magic number

File: src/accommodation/accommodation.service.ts [hunk 2]
```diff
@@ -809,4 +843,16 @@ export class AccommodationService {
 
     return orderingObjsArray;
   }
+
+  private isDefaultOptions(options: OrderAndFilterDto): boolean {
+    return (
```
Comment (bakhromov96) [inline src/accommodation/accommodation.service.ts:857]: @Zach13131 this part of code looks strange, maybe try this:
```
const {
    limit = CachingPagesLimit,
    minPrice = 0,
    maxPrice = AccommodationMaxPrice,
    minRooms = 0,
    maxRooms = AccommodationMaxRooms,
    minPeople = 0,
    maxPeople = AccommodationMaxPeople,
  } = options;

  return (
    limit === CachingPagesLimit &&
    minPrice === 0 &&
    maxPrice === AccommodationMaxPrice &&
    minRooms === 0 &&
    maxRooms === AccommodationMaxRooms &&
    minPeople === 0 &&
    maxPeople === AccommodationMaxPeople
  );
```

**Review by bakhromov96 — CHANGES_REQUESTED** [review]

**Review by gitkoshelew — APPROVED** [review]

**Review by bakhromov96 — APPROVED** [review]
LGTM!

File: src/accommodation/accommodation.service.ts (hunk 2 above)
Comment (Zach13131) [reply src/accommodation/accommodation.service.ts:857]: yea, this looks much better thanks)

**Pull request merged**